Sources/Protocols/MessagesLayoutDelegate.swift

```diff
@@ -103,12 +103,26 @@ public protocol MessagesLayoutDelegate: AnyObject {
     /// The default value returned by this method is `false`.
     func shouldCacheLayoutAttributes(for message: MessageType) -> Bool
 
+    func cellTopLabelHeight(for message: MessageType, at indexPath: IndexPath, in messagesCollectionView: MessagesCollectionView) -> CGFloat
```

Comment:
The reason why I wanted to do this PR first is that now we can't cache label sizes since they become dynamic.
This means we can probably only cache `MessageContainerView` sizes and not the full cell height. Doesn't matter though, this tradeoff is worth it 👌